src/main/java/lotto/model/LottoPurchaseAmount.java

```diff
@@ -0,0 +1,42 @@
+package lotto.model;
+
+import java.util.Objects;
+
+public class LottoPurchaseAmount {
+    private long amount;
```

Comment:
`amount` 상태 값은 한번 할당되면 값이 변경되지 않도록 변경된 것 같네요. 😃
final 키워드를 추가해주면 좋을 것 같아요. 😉

```suggestion
    private final long amount;
```